tests/functional/image-uploads-non-auth.js

```diff
@@ -213,14 +213,8 @@ registerSuite("Image Uploads (non-auth)", {
       return FunctionalHelpers.openPage(this, url, ".js-remove-upload")
         .execute(POSTMESSAGE_TEST_SQUARE)
         .sleep(1000)
-        .execute(() => {
-          document.querySelector(".js-image-upload").click();
-        })
         .execute(POSTMESSAGE_TEST_SQUARE)
         .sleep(1000)
-        .execute(() => {
-          document.querySelector(".js-image-upload").click();
-        })
         .sleep(1000)
```

Comment:
probably second `.sleep(1000)` could be removed?

Reply:
oops, thanks -- missed that!